extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -71,6 +79,47 @@ export class AmpStoryInteractiveSlider extends AmpStoryInteractive {
     this.sliderType_ = SliderType.PERCENTAGE;
   }
 
+  /** @override */
+  displayOptionsData(responseData) {
+    const average = this.calculateWeightedAverage_(responseData);
+    setImportantStyles(this.rootEl_, {'--average': average + '%'});
+  }
+
+  /**
+   * @private
+   * @param {!Array<!InteractiveOptionType>} responseData
+   * @return {number}
+   */
+  calculateWeightedAverage_(responseData) {
+    let numerator = 0;
+    let denominator = 0;
+    for (let i = 0; i < responseData.length; i++) {
+      numerator += responseData[i].index * responseData[i].count;
+      denominator += responseData[i].count;
+    }
+    if (denominator == 0) {
+      return 0;
+    }
+    return numerator / denominator;
+  }
+
+  /** @override*/
+  updateComponentWithData(data) {
+    this.optionsData_ = this.orderData_(data);
+    this.optionsData_.forEach((response) => {
+      if (response.selected) {
+        this.hasUserSelection_ = true;
+        this.updateStoryStoreState_(response.index);
```

Comment:
Let's not update the story store state, since this component doesn't interact with the results component